discord/handler.py

```diff
@@ -148,6 +148,15 @@ def get_token_for_athlete(id):
 
 
 def post_message(body):
+    # See https://developers.strava.com/docs/reference/#api-models-ActivityType
+    activity_colours = {
+        'Run': int('fc4c02', 16),  # orange
+        'Ride': int('66c2ff', 16),  # pale blue
+        'Hike': int('008000', 16),  # forest green
+        'RockClimbing': int('ff8000', 16),  # rock colour?
+        'default': int('cc0000', 16)  # red
```

Comment:
Also I'm thinking this should be the same as the run colour, since `fc4c02` is the Strava logo colour

Reply:
This way we know it's an unmapped type.

Reply:
TBH I'd rather have the default be the same as what it is now, then we can add more types in the future if required.

Reply:
Ok